whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+seeing, the scheduler will have a list of priorities for next-field, based on
+prior history of coverage.  The relevant parameters are seeing, depth, and
+camera rotation angle with respect to North and to zenith. Nearby fields in need
+of coverage in these bands should be given high priority if the seeing is better
+than some specified value, likely 0.7 arcsec FWHM \citep[Sec
+14.5.2]{2009arXiv0912.0201L}.
 
 \subsection{Target Measurements}
 
-It is expected that even after maximal optimization of camera optics
-and electronics, that systematic image shape errors will be associated
-with the orientation of the camera focal plane.  Using data from vendor CCDs, simulations 
+It is expected that even after maximal optimization of camera optics and
+electronics, that systematic image shape errors will be associated with the
+orientation of the camera focal plane.  Using data from vendor CCDs, simulations
 of LSST observing have shown that a combination of x-y dithering on the sky and
-pipline processing with pixel re-map (to cancel much of the CCD frame fixed
+pipeline processing with pixel re-map (to cancel much of the CCD frame fixed
 distortions) can get well within a factor of ten of the goal for shear
-systematics residuals.  Simulations which add camera angle dithering show
-that the goal can be achieved in fields with relatively uniform seeing history.
-
-Thus shear systematics will be partially reduced by randomization of the
-orientation of the camera with respect to the sky.  This is
-represented by the parameter RotSkyPos: we can construct diagnostic
-metrics that quantify the uniformity of its distribution at each sky
-position.   Given the spin 2 symmetry of shear, if the camera can rotate over a range of +/-
-90 deg, the optimal strategy for shear systematics suppression will be to aim for
-uniformity if the square of the cosine of the camera rotation angle.  i.e. +90 deg 
-and -90 deg are degenerate, so we want half as much coverage at those angles than at 0 deg.
-
-Similarly, the telescope optics may harbor systematic aberrations, and
-these also could be mitigated by recording images with varying
-parallactic angle.  Another relevant parameter, RotTelPos, is
-indicative of the projected angle of the telescope optics on the sky. More 
-important is the effect of atmospheric chromatic diffraction: re-visits to a given
-field should be distributed over RA, consistent with airmass and seeing limits.
-
-Uniformity of depth is important, but less so than uniformity in camera
-rotator shear suppression.  Simulations have shown that for the Gold sample of galaxies,
-uniformity at the 0.2 mag level in limiting magnitude produces little shear bias. The
-largest effect comes from bias in weak lens magnification tomography.
+systematics residuals.  Simulations which add camera angle dithering show that
+the goal can be achieved in fields with relatively uniform seeing history
```

Comment:
To average down the PSF systematics over many re-visits we benefit from 
uniformly distributed image quality over the ensemble.  A non-uniform 
history in some field can be addressed by the scheduler taking that into 
account for the offending rotation angle(s) in the history of prior visits.

On 6/29/16 6:51 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69058489>:
>
>> -uniformity if the square of the cosine of the camera rotation angle.  i.e. +90 deg
>> -and -90 deg are degenerate, so we want half as much coverage at those angles than at 0 deg.
>> -
>> -Similarly, the telescope optics may harbor systematic aberrations, and
>> -these also could be mitigated by recording images with varying
>> -parallactic angle.  Another relevant parameter, RotTelPos, is
>> -indicative of the projected angle of the telescope optics on the sky. More
>> -important is the effect of atmospheric chromatic diffraction: re-visits to a given
>> -field should be distributed over RA, consistent with airmass and seeing limits.
>> -
>> -Uniformity of depth is important, but less so than uniformity in camera
>> -rotator shear suppression.  Simulations have shown that for the Gold sample of galaxies,
>> -uniformity at the 0.2 mag level in limiting magnitude produces little shear bias. The
>> -largest effect comes from bias in weak lens magnification tomography.
>> +systematics residuals.  Simulations which add camera angle dithering show that
>> +the goal can be achieved in fields with relatively uniform seeing history
>
> What exactly is the connection to a uniform seeing history here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69058489>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6Ypx-iKuhreuIlPQPwGOLH01vqT-9ks5qQyEtgaJpZM4I9hd->.
>